src/pages/signUpPage.ts

```diff
@@ -0,0 +1,30 @@
+import { Locator, Page } from '@playwright/test';
+import PlaywrightWrapper from '../helper/wrapper/playwrightWrappers';
+
+export default class SignUpPage {
+  private base: PlaywrightWrapper;
+
+  constructor(private page: Page) {
+    this.base = new PlaywrightWrapper(page);
+  }
+  fullNameInput = this.page.locator("(//input[@class='el-input__inner'])[1]");
+  companyNameInput = this.page.locator("(//input[@class='el-input__inner'])[2]");
+  companyEmailInput = this.page.locator("(//input[@class='el-input__inner'])[3]");
+  confirmEmailInput = this.page.locator("//input[@data-element='email_confirm']");
+  country = this.page.locator("(//i[contains(@class,'el-select__caret el-input__icon')])[1]");
```

Comment:
what does the name `country` tell about this variable/property? What is the user expected to get into mind when he or she reads this name?